com.unity.multiplayer.mlapi/Runtime/Serialization/Pooled/BitStreamPool.cs

```diff
@@ -9,10 +9,13 @@ namespace MLAPI.Serialization.Pooled
     /// </summary>
     public static class BitStreamPool
     {
-        private static byte createdStreams = 0;
+        private static uint createdStreams = 0;
         private static readonly Queue<WeakReference> overflowStreams = new Queue<WeakReference>();
         private static readonly Queue<PooledBitStream> streams = new Queue<PooledBitStream>();
 
+        private const uint MaxBitPoolStreams = 1024;
+        private const uint MaxCreatedDelta = 768;
```

Comment:
I think changes in this file should be under a separate PR and we need to look at BitStream/Reader/Writer and Pool implementations separately.

Reply:
I agree with you 90%.  These changes were needed to exceed the previously hardcoded values.  Some of the modifications made to those (and other) areas are temporary fixes for some areas in MLAPI that could be better optimized.  Many of the modifications in the aforementioned regions were used more as tools to determine what kind of an impact the added RPC queue feature had for both GC Alloc and frame time.  This chain of events arose during my GC Alloc pass to assure we stayed as close to a 0 GC Alloc footprint as possible (within alpha feature boundaries) when I noticed areas of the RPC Queue code base were taking up too many cycles and had to determine whether it was the initial implementation or if it was some other area that could be further optimized.
Either case, we would need to refactor any code that we wanted to move to a different PR and test the changes before pushing them back to this current PR.  However, that could take more time than if we were to take note of all BitStream/Reader/Writer and pool related changes and adding them as "areas of interest" for the task that would drive the suggested new PR.

My personal preference would be the later scenario.  :)